lib/traject/import_helper.rb

```diff
@@ -1,61 +1,11 @@
 # frozen_string_literal: true
 
-# rubocop:disable Style/ClassVars (OK here since we are not inheriting this class)
-# https://rubydoc.info/gems/rubocop/RuboCop/Cop/Style/ClassVars
```

Comment:
The code that was here to find the leader in the Solr replication is not needed anymore now that we are using a new collection to perform the index.